vllm/v1/core/scheduler.py

```diff
@@ -223,6 +294,51 @@ def _make_running_request_data(
             self.running_reqs_data[request.request_id] = req_data
         return req_data
 
+    def _schedule_encoder_inputs(
+        self,
+        request: Request,
+        num_computed_tokens: int,
+        num_new_tokens: int,
+        encoder_budget: int,
+    ) -> Tuple[List[int], int]:
```

Comment:
Please docstring this function for readability.

Reply:
Added. Thanks for the suggestion.